src/plugins/kibana_usage_collection/server/collectors/management/schema.ts

```diff
@@ -73,6 +73,7 @@ export const stackManagementSchema: MakeSchemaFrom<UsageStats> = {
   'discover:sort:defaultOrder': { type: 'keyword' },
   'context:step': { type: 'long' },
   'accessibility:disableAnimations': { type: 'boolean' },
+  'fileUpload:maxFileSize': { type: 'keyword' },
```

Comment:
Should we remove `ml:fileDataVisualizerMaxFileSize` from the schema/types, or would it break previously collected values @afharo?

Reply:
This was asked on slack, i'll just reflect it here:

> We should eemove the old one. `tslint`  and the `telemetry_check` should complain otherwise (CI fails)

> The schema is versioned. If that field won’t exist/be reported anymore in 7.13, it should be removed